### PR TITLE
New threat hunting playbooks for AutoFocus

### PR DESCRIPTION
**Pull request by TalNos**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5419

## Description
Create new playbooks for AutoFocus for the purpose of threat hunting. The new playbooks will be part of the 'Palo Alto Networks - Hunting And Threat Detection' playbook.

## Screenshots
'Autofocus - File Indicators Hunting' playbook:
Autofocus_-_File_Indicators_Hunting

'Autofocus - Traffic Indicators Hunting' playbook:
Autofocus_-_Traffic_Indicators_Hunting

'Autofocus - Hunting And Threat Detection' playbook:
Autofocus_-_Hunting_And_Threat_Detection

### CODE REVIEW

File: Packs/AutoFocus/Playbooks/playbook-Autofocus_-_File_Indicators_Hunting_README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+A simple search mode is used to query Autofocus based on the file indicators specified in the playbook inputs. Advanced search mode queries can also be used with multiple query parameters, but require all field names, parameters, and operators (JSON format) to be specified. 
+It is recommended to use the Autofocus UI to create an advanced query, export it and paste it into the relevant playbook inputs.
+
+Note that multiple search values should be separated by commas only (without spaces or any special characters).
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+* Autofocus Query Samples, Sessions and Tags_copy
```
Comment (altmannyarden) [inline Packs/AutoFocus/Playbooks/playbook-Autofocus_-_File_Indicators_Hunting_README.md:12]: Need to fix that

File: Packs/AutoFocus/Playbooks/playbook-Autofocus_-_Traffic_Indicators_Hunting_README.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+A simple search mode is used to query Autofocus based on the traffic indicators specified in the playbook inputs. Advanced search mode queries can also be used with multiple query parameters, but require all field names, parameters, and operators (JSON format) to be specified. 
+It is recommended to use the Autofocus UI to create an advanced query, export it and paste it into the relevant playbook inputs.
+
+Note that multiple search values should be separated by commas only (without spaces or any special characters).
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+* Autofocus Query Samples, Sessions and Tags_copy
```
Comment (altmannyarden) [inline Packs/AutoFocus/Playbooks/playbook-Autofocus_-_Traffic_Indicators_Hunting_README.md:12]: Need to fix that

Comment (altmannyarden) [reply Packs/AutoFocus/Playbooks/playbook-Autofocus_-_Traffic_Indicators_Hunting_README.md:12]: @TalNos

**Comment by julieschwartz18** [comment]
@TalNos - Doc review complete for the yaml files, updated descriptions throughout the file. Please confirm the updates and regenerate the README files from the yamls. Thanks!

File: Packs/AutoFocus/Playbooks/playbook-Autofocus_-_File_Indicators_Hunting_README.md (hunk 1 above)
Comment (TalNos) [reply Packs/AutoFocus/Playbooks/playbook-Autofocus_-_File_Indicators_Hunting_README.md]: already fixed yesterday morning.

File: Packs/AutoFocus/Playbooks/playbook-Autofocus_-_Traffic_Indicators_Hunting_README.md (hunk 2 above)
Comment (TalNos) [reply Packs/AutoFocus/Playbooks/playbook-Autofocus_-_Traffic_Indicators_Hunting_README.md]: done

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**